Задание_3/Задание_3/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Задание_3
+{
+    internal class Program
+    {
+
+
+        static void Main()
+        {
+            double x, x3, x2, Y, A;
+            Console.WriteLine("Введите значение x");
+            x = Convert.ToDouble(Console.ReadLine());
+            x3 = 0;
+            x2 = 0;
+            Y = 0;
+            A = 0;
+            nomber(x, ref x3, ref x2, ref Y, ref A);
+            Console.WriteLine($"Ответ: {Math.Round(Y, 3)}");
+
+        }
+
+
+        static void nomber(double x, ref double x3, ref double x2, ref double Y, ref double A)
```

Comment:
В методе много лишних параметров и он должен возвращать значение. Возращение значение через параметр, переданный по ссылке, здесь не оправдано сложно и запутано. Имя метода с большой буквы.
```suggestion
        static double Nomber(double x)
```